tasks/gen/resource/browser.cr

```diff
@@ -134,12 +134,24 @@ class Gen::Resource::Browser < LuckyCli::Task
 
   private def columns_are_valid? : Bool
     column_definitions.any? && column_definitions.all? do |column_definition|
-      column_parts = column_definition.split(":")
-      column_name = column_parts.first
+      column_parts = parse_definition(column_definition)
+      column_name = column_parts.first?.to_s
       column_parts.size == 2 && column_name == column_name.underscore
     end
   end
 
+  private def parse_definition(column_definition : String) : Array(String)
+    matcher = /^([a-z_0-9]+):(Bool|Int16|Int32|Int64|String|UUID|Time|Float64|JSON::Any|Array\(\w+\))/
```

Comment:
Regex is never pretty, but I'm not sure a cleaner way to do this. I originally wrote one that allowed for basically any constant, but I figure this might be a little more "type-safe"(ish) in the case that someone tries to do something like `x:Uint8` which we don't support. 

An alternative to parsing the definition this way could be to not allow `JSON::Any`, and just do something like `data:JSON`. But then we have to watch for that string specifically and append the `::Any` where necessary, and also add some sort of error message if someone does the full one. I feel like that would get just as messy and confusing though.